File: Web/RMA-Web-Shell/projects/clientcare/src/app/policy-manager/shared/entities/benefit-payroll.ts
import {
  BenefitPayrollStatusTypeEnum
} from "../../../../../../shared-models-lib/src/lib/enums/benefit-payroll-status-type-enum";

export class BenefitPayroll {
    rowId : number;
    benefitPayrollId: number;
    benefitDetailId: number;
    benefitCategory: string;
    rolePlayerId: number;
    policyId: number;
    effectiveDate: Date;
    linkedBenefitPayrollId: number;
    monthlySalary: number;
    sumAssured: number;
    noOfMembers: number;
    fixedSum: number;
    payrollStatusType: BenefitPayrollStatusTypeEnum;
    billingLevel : string
    billingLevelCode : string
    benefitCategoryId: number;
    billingRate: number;
    billingMethod : string;
    billingMethodCode : string;
    premiumDue : number;
    lastUpdatedDate : Date;
    fixedPremium : number;
    benefitId: number;

  }

File: Service/ClientCare/RMA.Service.ClientCare.Contracts/Entities/Policy/BenefitPayroll.cs

using RMA.Service.Admin.MasterDataManager.Contracts.Enums;
using System;

namespace RMA.Service.ClientCare.Contracts.Entities.Policy
{
    public class BenefitPayroll
    {
        public int BenefitPayrollId { get; set; }
        public int BenefitDetailId { get; set; }
        public int? BenefitCategoryId { get; set; }
        public int PolicyId { get; set; }
        public int RolePlayerId { get; set; }
        public System.DateTime EffectiveDate { get; set; }
        public int? LinkedBenefitPayrollId { get; set; }
        public decimal? MonthlySalary { get; set; }
        public decimal? SumAssured { get; set; }
        public int? NoOfMembers { get; set; }
        public decimal? FixedPremium { get; set; }
        public BenefitPayrollStatusTypeEnum PayrollStatusType { get; set; }
        public string BenefitCategory { get; set; }
        public string BillingLevel { get; set; }
        public string BillingLevelId { get; set; }
        public decimal BillingRate { get; set; }
        public string BillingMethod  { get; set; }
        public string BillingMethodCode  { get; set; }
        public decimal PremiumDue { get; set; }
        public int BenefitId  { get; set; }
        public DateTime LastUpdatedDate { get; set; }
    }
}
